connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -139,10 +139,23 @@ public Object run() {
     }
 
     protected void initLoaders() {
-        String path = null;
+        for (String configPath : pluginPaths) {
+            initPluginLoader(configPath);
+        }
+        // Finally add parent/system loader.
+        initPluginLoader("classpath");
```

Comment:
Might be nice to have this literal be a constant, since it's referenced in two places.

Reply:
Good point. Moved to member variable.